.env.example

```diff
@@ -2,4 +2,6 @@ OPENAI_API_KEY='<Optional. Open AI API key for AI help in Supabase studio>'
 VITE_SUPABASE_URL='http://127.0.0.1:54321'
 VITE_SUPABASE_ANON_KEY='eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0'
 SUPABASE_SERVICE_ROLE_KEY='eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImV4cCI6MTk4MzgxMjk5Nn0.EGIM96RAZx35lJzdJsyH-qQwv8Hdp7fsn3W0YpN81IU'
-SUPABASE_JWT_SECRET='super-secret-jwt-token-with-at-least-32-characters-long'
\ No newline at end of file
+SUPABASE_JWT_SECRET='super-secret-jwt-token-with-at-least-32-characters-long'
+VITE_OPEN_SECRET_CLIENT_ID='a5e351d4-6dd9-4ee6-8e36-f5fb871e5432'
```

Comment:
@gudnuf I created a new env for open secret for local development. Make sure to copy these new values to your .env file when this gets merged.